modules/rhcos-add-extensions.adoc

```diff
@@ -8,10 +8,14 @@
 = Adding extensions to {op-system}
 {op-system} is a minimal container-oriented RHEL operating system, designed to provide a common set of capabilities to {product-title} clusters across all platforms. While adding software packages to {op-system} systems is generally discouraged, the MCO provides an `extensions` feature you can use to add a minimal set of features to {op-system} nodes.
 
-Currently, the following extension is available:
+Currently, the following extensions are available:
 
 * **usbguard**: Adding the `usbguard` extension protects {op-system} systems from attacks from intrusive USB devices. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/security_hardening/index#usbguard_protecting-systems-against-intrusive-usb-devices[USBGuard] for details.
 
+* **libkrb5**: Adding the `libkrb5` extension provides the library routines for the Kerberos 5 library, a library that authenticates users to network services using secret-key cryptography. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/configuring_and_managing_identity_management/index[_Configuring and managing Identity Management_] for more information about Kerberos.
+
+* **krb5-workstation**: Adding the `krb5-workstation` extension provides an interface for the Kerberos 5 library and contains the `kinit`, `klist`, `kdestroy` and `kpasswd` Kerberos programs. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/deploying_different_types_of_servers/securing-nfs_deploying-different-types-of-servers#configuring-an-nfs-server-and-client-to-use-kerberos_securing-nfs[_Configuring an NFS server and client to use Kerberos_] for information about how to set up a Kerberized NFS server.
+
```

Comment:
I realized after having a sleep on things that these are the individual packages that are installed, however the name of the extension is `kerberos`

The design of the extensions allows us to capture multiple packages to be installed with a single identifier, such at this case.

See the MCO code where the `kerberos` identifier maps to the two packages - https://github.com/openshift/machine-config-operator/blob/377b4d23a452d20ed2a81792cebc70dba2e94f94/pkg/daemon/update.go#L1071

Reply:
Thank you! This makes figuring out the short description a bit easier, too. Fixed